src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,124 @@
+import org.testng.annotations.Test;
+
+public class Exercise1Test {
+
+    @Test
+    public void exercise1Check() {
+
+        //setup Chrome Driver
+        System.setProperty("webdriver.chrome.driver", "/absolute/path/to/binary/chromedriver");
+        WebDriverManager.chromedriver().setup();
+
+        //setup options for Chrome Driver
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--no-sandbox", "start-maximized");
+
+        WebDriver driver = new ChromeDriver(options);
+
+        //step 1 - navigate to the site
+        driver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        SoftAssertions softAssertions = new SoftAssertions();
+
+        //step 2 - assert browser title
+        softAssertions.assertThat(driver.getTitle()).contains("Page Home");
+
+        //step 3 - perform login
+        WebElement element = driver.findElement(By.id("user-icon"));
+        element.click();
+
+        element = driver.findElement(By.id("name"));
+        element.click();
+        element.sendKeys("Roman");
+
+        element = driver.findElement(By.id("password"));
+        element.click();
+        element.sendKeys("Jdi1234");
+
+        element = driver.findElement(By.id("login-button"));
+        element.click();
+
+        //step 4 - Assert Username is loggined
+        driver.findElement(By.id("user-name"));
+        softAssertions.assertThat(driver.getTitle()).isEqualTo("Roman Iovlev");
+
+        //Assert that there are 4 items on the header section are displayed
+        List<WebElement> webElementList = driver.findElements(By.xpath("//*[contains(@class,'m-l8')]/li/a"));
+        softAssertions.assertThat(webElementList.size()).isEqualTo(4);
+
+        // and they have proper texts
+        softAssertions.assertThat(webElementList.get(0).getText()
+                .toLowerCase(Locale.ROOT)).isEqualTo("home");
+        softAssertions.assertThat(webElementList.get(1).getText()
+                .toLowerCase(Locale.ROOT)).isEqualTo("contact form");
+        softAssertions.assertThat(webElementList.get(2).getText()
+                .toLowerCase(Locale.ROOT)).isEqualTo("service");
+        softAssertions.assertThat(webElementList.get(3).getText()
+                .toLowerCase(Locale.ROOT)).isEqualTo("metals & colors");
+
+        // Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> webElementListImages = driver.findElements(By.className("icons-benefit"));
+        softAssertions.assertThat(webElementListImages.size()).isEqualTo(4);
+
+        softAssertions.assertThat(webElementListImages.get(0).isDisplayed()).isEqualTo(true);
+        softAssertions.assertThat(webElementListImages.get(1).isDisplayed()).isEqualTo(true);
+        softAssertions.assertThat(webElementListImages.get(2).isDisplayed()).isEqualTo(true);
+        softAssertions.assertThat(webElementListImages.get(3).isDisplayed()).isEqualTo(true);
```

Comment:
In this case and all cases below is more convenient to use .isTrue() instead of comparing to 'true'